src/rpdk/core/contract/resource_generator.py

```diff
@@ -27,18 +28,28 @@
 # https://github.com/aws-cloudformation/aws-cloudformation-rpdk/issues/118
 
 # Arn is just a placeholder for testing
+# format list taken from https://python-jsonschema.readthedocs.io/en/stable/validate/#jsonschema.FormatChecker.checkers
+# date-time regex from https://github.com/naimetti/rfc3339-validator
+# date is extraction from date-time
+# time is extraction from date-time
 STRING_FORMATS = {
     "arn": "^arn:aws(-(cn|gov))?:[a-z-]+:(([a-z]+-)+[0-9])?:([0-9]{12})?:[^.]+$",
```

Comment:
from here should we be adding the rest of the formats? 
https://json-schema.org/understanding-json-schema/reference/string.html#id8

Missing ipv4, ipv6, hostname, and the internationalized versions.  We can table it as they come up just curious

Reply:
We should add in the rest that are supported by cloudformation.  Let me create an issue for tracking about it.  The immediate fix was to stop the errors from the examples provided.

Reply:
We should probably also move away from the .example setup since hypothesis docs recommends the @given syntax.